internal/otelcollector/config/metric/shared_attributes.go

```diff
@@ -12,3 +8,21 @@ const (
 	InputSourceIstio      InputSourceType = "istio"
 	InputSourceOtlp       InputSourceType = "otlp"
 )
+
+const (
+	TransformedInstrumentationScopeRuntime    = "io.kyma-project.telemetry/runtime"
+	TransformedInstrumentationScopePrometheus = "io.kyma-project.telemetry/prometheus"
+	TransformedInstrumentationScopeIstio      = "io.kyma-project.telemetry/istio"
+)
+
+var TransformedInstrumentationScope = map[InputSourceType]string{
+	InputSourceRuntime:    TransformedInstrumentationScopeRuntime,
+	InputSourcePrometheus: TransformedInstrumentationScopePrometheus,
+	InputSourceIstio:      TransformedInstrumentationScopeIstio,
+}
+
+var UpstreamInstrumentationScopeName = map[InputSourceType]string{
```

Comment:
Does it have to be here? Isn't it an internal detail, only the metric agent should know about?

Reply:
If you move it there, you could simplify the other constants:
TransformedInstrumentationScopeRuntime -> InstrumentationScopeRuntime

Reply:
fixed